src/_help/getting-started/upload-your-first-definition.md

```diff
@@ -15,23 +15,24 @@ Two options are available: using your email address and creating a username, or
 
 ![](/images/help/signup.png)
 
-Pay attention when choosing your username: it will act as a slug to access your API documentation and will appear in the URL (unless you configure a custom domain later). You can change this username/slug later on.
-
 ## Create your first documentation
 
-### Step 1: Create your documentation
+### Step 1: Create your organization
+
+Now that your Bump.sh account is created, you will immediately be prompted to create your first organization.
+An organization allows you to work alone or collaboratively with your API documentation and hubs (a collection of documentation) within the same workspace.
```

Comment:
The with + within surprised me here, but maybe it's a mistake from me.

Just to be sure, here is a suggestion:
```suggestion
An organization allows you to work alone or collaboratively on your API documentation and hubs (a collection of documentation) within the same workspace.
```